buildSrc/src/main/groovy/io/micronaut/guides/GuideAsciidocGenerator.groovy

```diff
@@ -166,9 +165,7 @@ class GuideAsciidocGenerator {
             if (rawLine.startsWith(CALLOUT) && rawLine.endsWith(']')) {
                 rawLines << callout(rawLine, projectDir)
             } else if (rawLine.startsWith(INCLUDE_COMMONDIR) && rawLine.endsWith('[]')) {
-                include rawLine, rawLines, projectDir, true
-            } else if (rawLine.startsWith(EXTERNAL) && rawLine.endsWith(']')) {
-                include rawLine, rawLines, projectDir, false
+                include(rawLine, rawLines, projectDir)
```

Comment:
I think we should use parenthesis in general when invoking methods. Not sure what is your opinion about this.